versioned_docs/version-22.04/update/update-centreon-platform.md

```diff
@@ -32,7 +52,31 @@ Then upgrade all the components with the following command:
 yum update centreon\*
 ```
 
-### Finalizing the update
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+Clean the cache:
+
+```shell
+apt clean all
+apt update
+
+```
+
+Then upgrade all the components with the following command:
+
+```shell
+apt upgrade centreon\*
+```
+
+</TabItem>
+</Tabs>
+
+> Now you can either perform the update:
+> - [Using the wizard](#finalize-the-update-using-wizard)
+> - [Using a dedicated API endpoint](#finalize-the-update-using-api-endpoint)
```

Comment:
Now you can either perform the update: 
- [Using the wizard](#finalize-the-update-using-the-wizard)
- [Using a dedicated API endpoint](#finalize-the-update-using-the-api-endpoint)